src/OrchardCore.Modules/OrchardCore.Tenants/Views/Admin/Index.cshtml

```diff
@@ -129,67 +129,74 @@
             @foreach (var entry in Model.ShellSettingsEntries)
             {
                 <li class="list-group-item">
-                    <div class="custom-control custom-checkbox float-left">
-                        <input type="checkbox" class="custom-control-input" value="@entry.Name" name="tenantNames" id="tenantNames-@entry.Name">
-                        <label class="custom-control-label" for="tenantNames-@entry.Name"></label>
-                    </div>
-                    <div class="properties">
-                        <div class="related">
-                            <a asp-action="Edit" asp-route-id="@entry.Name" class="btn text-primary btn-sm" data-toggle="tooltip" title="@T["Edit"]"><i class="fa fa-edit"></i></a>
-                            @if (entry.ShellSettings.State == TenantState.Running)
-                            {
-                            <a class="btn text-info btn-sm" href="@GetDisplayUrl(entry, originalPathBase)" target="_blank" data-toggle="tooltip" title="@GetDisplayUrl(entry, originalPathBase)"><i class="fa fa-external-link"></i></a>
-                            }
-                            @if (!entry.IsDefaultTenant && entry.ShellSettings.State == TenantState.Running)
-                            {
-                                <a asp-action="Disable" asp-route-id="@entry.Name" class="btn text-danger btn-sm" itemprop="UnsafeUrl" data-toggle="tooltip" title="@T["Disable"]"><i class="fa fa-toggle-off"></i></a>
-                            }
-                            @if (entry.ShellSettings.State == TenantState.Disabled)
-                            {
-                                <a asp-action="Enable" asp-route-id="@entry.Name" class="btn text-success btn-sm" itemprop="UnsafeUrl" data-toggle="tooltip" title="@T["Enable"]"><i class="fa fa-toggle-on"></i></a>
-                            }
-                            @if (entry.ShellSettings.State == TenantState.Uninitialized)
-                            {
-                                <a class="btn text-info btn-sm" id="btn-setup-@entry.Name" href="@GetEncodedUrl(entry, originalPathBase)" data-toggle="tooltip" title="@T["Setup"]"><i class="fa fa-cog"></i></a>
-                            }
-                            <a asp-action="Reload" asp-route-id="@entry.Name" class="btn text-secondary btn-sm" itemprop="UnsafeUrl" data-toggle="tooltip" title="@T["Reload"]"><i class="fa fa-redo"></i></a>
+                    <div class="row">
+                        <div class="col-xl-6 col-sm-12 title">
```

Comment:
col-lg-6 would be sufficient

Reply:
I tried quickly with Flexboxes and I've not found a perfect solution with it yet. Quicker with a BS grid.